ds_import/face.py

```diff
@@ -28,14 +18,27 @@ def vertices(self):
 
     @vertices.setter
     def vertices(self, vertices):
-        self._vertices = vertices
+
+        # Verify the list is exactly 3 elements long
+        if len(vertices) != 3:
+            raise ValueError('List of three unsigned integers expected.')
+        else:
+            # Verify each element in the list is an unsigned int
+            for vertex_index in vertices:
+                if not isinstance(vertex_index, int):
+                    raise TypeError('Objects of type int expected, however type {} was passed'.format(type(vertex_index)))
+                else:
+                    if vertex_index < 0:
+                        raise ValueError('Expecting unsigned ints')
+                    else:
+                        self._vertices = vertices
+
+    # TODO: Implement vertex order methods below.
 
     def blender_ordered_vertices(self):
         """Returns the vertices in an order that Blender expects."""
-        # TODO
         pass
 
     def darksouls_ordered_vertices(self):
         """Returns the vertices in an order that Dark Souls expects."""
-        # TODO
         pass
```

Comment:
If we need to reorder anything it will be the face indices.

Anyway, I changed my tune and the extractor currently stores them in a Blender-wise fashion so this may not be necessary at all anywhere.  Thoughts?

Reply:
I'll remove the clutter.